docroot/themes/contrib/civictheme/includes/node.inc

```diff
@@ -107,61 +107,3 @@ function _civictheme_preprocess_page__node(&$variables) {
   // Hide sidebar check.
   $variables['hide_sidebar'] = civictheme_get_field_value($node, 'field_c_n_hide_sidebar') ?? FALSE;
 }
-
```

Comment:
Removed this pre-processor to follow standard 
civictheme_preprocess_paragraph__civictheme_slider_slide_ref

New template:  [paragraph--civictheme-slider-slide-ref.html.twig](https://github.com/salsadigitalauorg/civictheme_source/pull/531/files#diff-2f0f1d63fdc5cd60d72eb837c37e49cefcc1a39321e352dd0d856fb91381d0d6)